vision/classification/efficientnet-lite4/README.md

```diff
@@ -0,0 +1,121 @@
+# EfficientNet-Lite4
+
+## Use Cases
+EfficientNet-Lite4 is an image classification model that achieves state-of-the-art accuracy. It is designed to run on mobile CPU, GPU, and EdgeTPU devices, allowing for applications on mobile and loT, where computational resources are limited.
+
+## Description
+EfficientNet-Lite 4 is the largest variant and most accurate of the set of EfficientNet-Lite model. It is an integer-only quantized model that produces the highest accuracy of all of the EfficientNet models. It achieves 80.4% ImageNet top-1 accuracy, while still running in real-time (e.g. 30ms/image) on a Pixel 4 CPU. 
+
+## Model
+
+ |Model        |Download | Download (with sample test data)|ONNX version|Opset version|
+|-------------|:--------------|:--------------|:--------------|:--------------|
+|EfficientNet-Lite4     | [51.9 MB](model/efficientnet-lite4.onnx)	  | [157 MB](efficientnet-lite4.tar.gz)|1.7.0|11|
+
+### Source
+Tensorflow => Onnx
+
+<hr>
+
+
+## Inference
+
```

Comment:
It would be nice if we could include an inference notebook here as well. See Jennifer's PR for an example: https://github.com/onnx/models/pull/322 . Alternatively, you could just pull the relevant inference code (load into onnxruntime, pass in inputs / outputs) from the TF-onnx tutorial notebook and copy-paste it in this readme.